util/topgen/templates/chiplevel.sv.tpl

```diff
@@ -1018,7 +1018,8 @@ module chip_${top["name"]}_${target["name"]} (
   logic [rstmgr_pkg::PowerDomains-1:0] por_n;
   assign por_n = {ast_pwst.main_pok, ast_pwst.aon_pok};
   top_${top["name"]} #(
-    .PinmuxAonTargetCfg(PinmuxTargetCfg)
+    .PinmuxAonTargetCfg(PinmuxTargetCfg),
+    .UsbdevRcvrWakeCycles(4800)
```

Comment:
actually you don't need to hard wire this here. 
you can actually put a param declaration in `top_earlgrey.hjson` (see https://github.com/lowRISC/opentitan/blob/master/hw/top_earlgrey/data/top_earlgrey.hjson#L585), that way the hjson will be the source of truth for the taped out design, and we can override for other non-asic platforms.